### PR TITLE
[WIP] added lightweight dependency form

### PR DESCRIPTION
**Pull request by ar4s**

### CODE REVIEW

File: src/ralph/static/src/js/autofill.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+(function ($) {
+    'use strict';
+    var connections = [
```
Comment (szok) [inline src/ralph/static/src/js/autofill.js:3]: :+1:

File: src/ralph/admin/mixins.py [hunk 2]
```diff
@@ -41,10 +41,12 @@ def get_common_media():
         ('admin', 'js', 'actions.js'),
         ('admin', 'js', 'admin', 'RelatedObjectLookups.js'),
         ('vendor', 'js', 'jquery.js'),
+        ('src', 'js', 'ralph.init.js'),
         ('vendor', 'js', 'foundation.min.js'),
         ('vendor', 'js', 'modernizr.js'),
-        ('src', 'js', 'fill-fields.js'),
         ('vendor', 'js', 'foundation-datepicker.js'),
+        # ('src', 'js', 'fill-fields.js'),
```
Comment (mkurek) [inline src/ralph/admin/mixins.py]: what about it?

File: src/ralph/static/src/js/autofill.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        {
+            app: 'back_office', model: 'backofficeasset',
```
Comment (mkurek) [inline src/ralph/static/src/js/autofill.js]: wouldn't it be better to move model to next line?

File: src/ralph/static/src/js/autofill.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+            conf: [
+                {
+                    src: 'user',
+                    func: function($form, $src, val) {
+                        $('[name=owner]').val(val);
```
Comment (mkurek) [inline src/ralph/static/src/js/autofill.js]: what if i have matching input outside this form? shouldn't scope of this function be limited only to form? in other words, shouldn't you filter only inside passed form?

**Comment by mkurek** [comment]
:ok:

**Pull request closed without merging**